x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_requirements_page/es_requirements_page.tsx

```diff
@@ -55,8 +55,8 @@ export const MissingESRequirementsPage: React.FunctionComponent<{
 
   return (
     <WithoutHeaderLayout>
-      <EuiPageBody restrictWidth={820}>
-        <EuiPageContent hasBorder={true} hasShadow={false}>
+      <EuiPageBody restrictWidth={820} style={{ border: '1px solid #d3dae6', borderRadius: '5px' }}>
```

Comment:
Could we move the style to `styled-components`? We could use an enum from eui for the color if there is one.